File: Selenium/Activity2.java
package com.in;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity2 {

	public static void main(String[] args) throws InterruptedException {
		WebDriver driver = new FirefoxDriver();
		driver.get("https://training-support.net/webelements/login-form/");
		System.out.println("Tile is : "+driver.getTitle());
		driver.findElement(By.xpath("//input[@type= 'text'][@name='username']")).sendKeys("admin");
		driver.findElement(By.xpath("//input[@type= 'password'][@name='password']")).sendKeys("password");
		driver.findElement(By.xpath("//button[@class= 'svelte-1pdjkmx']")).click();
	    driver.quit();
		
	}
}

File: Selenium/Activity2.py
# Import webdriver from selenium
from selenium import webdriver
from selenium.webdriver.common.by import By

# Start the Driver
with webdriver.Firefox() as driver:
    # Navigate to the URL
    driver.get("https://training-support.net/webelements/login-form/")

    # Print the title of the page
    print("Page title is: ", driver.title)
    driver.find_element(By.XPATH,"//input[@type= 'text'][@name='username']").send_keys("admin")
    driver.find_element(By.XPATH,"//input[@type= 'password'][@name='password']").send_keys("password")
    driver.find_element(By.XPATH,"//button[@class= 'svelte-1pdjkmx']").click()
    print("Message",driver.find_element(By.CSS_SELECTOR, "h1.text-center").text)
    driver.quit()
